ui/pages/unlock-page/unlock-page.component.js

```diff
@@ -176,7 +178,15 @@ export default class UnlockPage extends Component {
               </div>
             ) : null}
           </div>
-          <h1 className="unlock-page__title">{t('welcomeBack')}</h1>
+          <Text
+            data-testid="unlock-page-title"
+            variant={TextVariant.headingLg}
+            marginTop={5}
+            fontWeight={800}
```

Comment:
We may need to figure out a way to map all of static styles to utility style props 🤔 
marginTop={1} = 4px
fontWeight={FontWeight.Bold} = 800 (headingLg is bold already so we can remove it)
```suggestion

```suggestion
            marginTop={1}
```

Reply:
[Removed fontWeight and updated margin](https://github.com/MetaMask/metamask-extension/pull/25227/commits/771701192adc3cdf36865374ccaddcccdd0b2917)